deps/meson.build

```diff
@@ -0,0 +1,9 @@
+subdir('blake2')
+subdir('file-updater')
+subdir('glad')
+if not jansson_dep.found()
+  subdir('jansson')
+endif
```

Comment:
This is what the first version of this branch did, but unfortunately it didn't work because flatpak-builder disables networking during the build step, and the project is only configured via meson during the build step :slightly_frowning_face:

Reply:
Is there a stage where you can run arbitrary commands to download the sources?

If so, `meson wrap download`.

If not, list the urls from the wrap file as additional manifest sources, and copy those files into `subprojects/packagecache/` -- meson only hits the network in order to download the files to the packagecache, if they are not there already, and either way it then extracts the subproject from the cached tarball the first time it is needed.

Reply:
I think the correct path forward is to add the jansson wrap, and also add the jansson module to flatpak-builder, so that Meson skips the download

Reply:
Sounds reasonable, I just have little practical knowledge of Flatpak other than that it exists. :D

Reply:
https://github.com/GeorgesStavracas/obs-gtk/pull/5 does that now, thanks!